src/app/about-us/page.tsx

```diff
@@ -1,62 +1,66 @@
-import imgvision1 from "@/lib/assets/vision-1.png";
-import imgvision2 from "@/lib/assets/vision-2.png";
 import imgapproach from "@/lib/assets/approach-11.png";
 import imgfakeNews from "@/lib/assets/fake-news.png";
-import imghandsCircle from "@/lib/assets/manos-circulo.png";
 import imgholdingWorld from "@/lib/assets/hombre-negocios-mundo-mano-1.png";
+import imghandsCircle from "@/lib/assets/manos-circulo.png";
+import imgvision1 from "@/lib/assets/vision-1.png";
+import imgvision2 from "@/lib/assets/vision-2.png";
 import Image from "next/image";
 
 export default function AboutUs() {
   return (
-    <main className="Rectangle198 min-h-screen bg-gradient-to-b from-[#a8dadc] via-design-light to-white   pt-[120px]">
-      <section className="w-[87%] max-w-[966px] mx-auto ">
-        <section className="mx-auto min-w-min max-w-max">
-          <div className="flex flex-row gap-[2%] pb-8 justify-center">
+    <main className="Rectangle198 min-h-screen bg-gradient-to-b from-[#a8dadc] via-design-light to-white pt-[120px]">
+      <section className="mx-auto mt-14 px-20">
+        <section className="mx-auto ">
+          <div className="grid grid-cols-[1fr_2fr] gap-6 pb-10">
             <Image
```

Comment:
Tratar de usar los valores q trae tailwind para mantener un estandar en el espaciado
```suggestion
          <div className="flex flex-row gap-10 pb-8 justify-center">
```